src/utils/react/useViewportSize.ts

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+
+interface ViewportSize {
+  width: number;
+  height: number;
+}
+
+// @ts-ignore
+let visualViewport = typeof window !== 'undefined' && window.visualViewport;
+
+export function useViewportSize(): ViewportSize {
+  let [size, setSize] = useState(() => getViewportSize());
```

Comment:
Seems if you dont need ‘let’ if you dont modify it

Reply:
it's was just copied from React Aria. No need to change anything I think :)